src/screens/Asset/ConfirmRWA.tsx

```diff
@@ -0,0 +1,222 @@
+import SavingIcon from "@assets/icons/saving.svg";
+import { BottomSheetModal } from "@gorhom/bottom-sheet";
+import { Currency } from "@notuslabs/wallet";
+import { Image } from "expo-image";
+import { router } from "expo-router";
+import { useAtom, useAtomValue } from "jotai";
+import { useCallback, useRef, useState } from "react";
+import { View, StyleSheet } from "react-native";
+
+import Container from "@/components/Container";
+import BottomSheet from "@/components/bottomSheet/BottomSheetModal";
+import Button from "@/components/buttons/Button";
+import Divider from "@/components/divider/Divider";
+import InvestimentReceipt from "@/components/receipts/InvestimentReceipt";
+import Heading from "@/components/typography/Heading";
+import Label from "@/components/typography/Label";
+import { borderRadius, dark, base, spacing } from "@/constants/DesignTokens";
+import { SIMPLEFORMAT } from "@/constants/configurarions";
+import { backedAssets } from "@/constants/mock";
+import useInvestRWA from "@/hooks/useInvestRWA";
+import { amountInputAtom } from "@/lib/atoms";
+import TransactionSuccess from "@/screens/TransactionSuccess";
+
+type ConfirmRWAProps = {
+  asset: string;
+};
+
+type SendTransactionType = {
```

Comment:
Again, **you don't have to change if you don't agree with me!**, but I would remove the suffix `Type`

```suggestion
type SendTransactionOptions = {
```